storage/byodb.md

```diff
@@ -52,3 +52,29 @@ The dynamic backends feature requires you to have one Snowflake warehouse for ea
 
 {: .image-popup}
 ![Snowflake account schema](schema.png)
+
+## BigQuery
+
+
+## BigQuery limitations
+
+Google Cloud Computing services, which include BigQuery, have introduced several limitations on their services. The following limitations are significant from the perspective of Keboola, particularly for owners of Keboola storage accounts. These limitations directly impact Keboola services and require management.
+
+Some of these limits are flexible (soft limits), depending on your contract with Google. This determines if the limit can be adjusted and by how much.
+
+**Number of Projects**
+
+Upon reaching **20** projects, you need to submit a ticket on GCP. Based on this submission, the number of projects can either be increased or the request may be denied.
+
+The Keboola application generates one project in GCP for each project within Keboola. Additionally, there is one main project overseeing all others. Therefore, you can have 19 projects in the Keboola application (+1 reserved for management).
+
+**Number of Service Accounts**
+
+Service accounts serve various purposes. Each project contains at least one service user. For instance, each user is created per workspace/sandbox, which technically limits the creation of more than 99 workspaces simultaneously. [Workspaces](https://help.keboola.com/transformations/workspace/) are employed for numerous tasks in Keboola, such as requiring a dedicated workspace for each transformation.
```

Comment:
```suggestion
Service accounts serve various purposes. Each project contains at least one service user. For instance, the is a user created per workspace/sandbox. This technically limits the existence of more than 99 workspaces at one time. [Workspaces](https://help.keboola.com/transformations/workspace/) are used for numerous tasks in Keboola. For example a dedicated workspace is created for each transformation. Note that it's deleted afterwards, so it counts toward the limit only during the runtime of the transformation job. 
```

Reply:
muze byt , jen si teda nejsem jisty tou prvni zmenou, ale to muze byt jen moji urovni anglictiny `For instance, **the is a user created** per workspace/sandbox.` . je to typo a nemelo by to byt spis `there is` 

@tomasfejfar

Reply:
"there" ✅